install_config/router/default_haproxy_router.adoc

```diff
@@ -68,8 +68,32 @@ routers deployment configuration and you may want to increase them based on the
 load of the router.
 ====
 
+[[deploy-router-create-router]]
+== Creating a Router
+
+The
+xref:../../install_config/install/quick_install.adoc#install-config-install-quick-install[quick
+installation] process automatically creates a default router. To create a router
```

Comment:
Maybe change "To create a router..." →  "Run the following command to create a router..."